jib-cli/src/test/java/com/google/cloud/tools/jib/cli/cli2/JibCliTest.java

```diff
@@ -0,0 +1,328 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.cloud.tools.jib.cli.cli2;
+
+import static com.google.common.truth.Truth.assertThat;
+import static com.google.common.truth.Truth8.assertThat;
+import static org.junit.Assert.fail;
+
+import com.google.cloud.tools.jib.api.Credential;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.Collection;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import picocli.CommandLine;
+
+public class JibCliTest {
+  @Test
+  public void testParse_missingRequiredParams() {
+    try {
+      CommandLine.populateCommand(new JibCli(), "");
+      fail();
+    } catch (CommandLine.MissingParameterException mpe) {
+      assertThat(mpe.getMessage()).isEqualTo("Missing required option: '--target=<target-image>'");
+    }
```

Comment:
How about doing something like this: 
```suggestion
    MissingParameterExceptin mpe = assertThrows(MissingParameterException.class, () ->  CommandLine.populateCommand(new JibCli(), ""));
    assertThat(mpe).hasMessageThat().contains("Missing required option: '--target=<target-image>'");
```

Reply:
oh interesting, new in junit 4.13?